src/entity/Keyword.ts

```diff
@@ -12,9 +12,13 @@ export type KeywordFlags = Partial<{
   onboarding: boolean;
   title: string;
   description: string;
+  roadmap: string;
 }>;
 
-export type KeywordFlagsPublic = Pick<KeywordFlags, 'title' | 'description'>;
+export type KeywordFlagsPublic = Pick<
+  KeywordFlags,
+  'title' | 'description' | 'roadmap'
```

Comment:
Just update the tests as well 🙏

Reply:
https://github.com/dailydotdev/daily-api/blob/10896baa0d508cd4e2fe52064d916bb0a0dc3e43/__tests__/keywords.ts#L362